recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "anaconda-client" %}
-{% set version = "1.6.13" %}
+{% set version = "1.6.14" %}
 {% set checksum = "2e47801beed0daf5b638707502b9976ea82fc63d98ea10a83814946fdd8a963c" %}
```

Comment:
Checksum needs updating too.

Reply:
yeah was basically using circle to fetch it for me :P